src/cloud/benchflow/sender/sender.go

```diff
@@ -329,25 +362,45 @@ func checkForErrors(errLog string) bool {
 	return false
 	}
 
+// Main function, which registers configurations and starts the consumers
 func main() {
-	kafkaIp = os.Getenv("KAFKA_IP")
-	kafkaPort = os.Getenv("KAFKA_PORT")
-	sparkMaster = os.Getenv("SPARK_MASTER")
-	sparkHome = os.Getenv("SPARK_HOME")
-	cassandraHost = os.Getenv("CASSANDRA_IP")
-	minioHost = os.Getenv("MINIO_IP")
+	// Settings for viper
+	viper.SetConfigName("config")
+	viper.AddConfigPath("/app/")
+	viper.AddConfigPath("./")
+	viper.AutomaticEnv()
+	err := viper.ReadInConfig()
+	if err != nil {
+	    panic(fmt.Errorf("Fatal error config file: %s \n", err))
+	}
+	
+	// Getting app configuration with viper, uses ENV Variables instead of set
+	cassandraKeyspace = viper.GetString("cassandra_keyspace")
+	kafkaIp = viper.GetString("kafka_host")
+	kafkaPort = viper.GetString("kafka_port")
+	cassandraHost = viper.GetString("cassandra_host")
+	minioHost = viper.GetString("minio_host")
+	minio_port = viper.GetString("minio_port")
+	sparkHome = viper.GetString("spark_home")
+	sparkMaster = viper.GetString("spark_master")
+	spark_port = viper.GetString("spark_port")
+	alluxio_port = viper.GetString("alluxio_port")
+	pysparkCassandraVersion = viper.GetString("pyspark_cassandra_version")
+	analysersPath = viper.GetString("analysers_path")
+	transformersPath = viper.GetString("transformers_path")
 	
-	dat, err := ioutil.ReadFile("configuration/config2.yml")
+	// Getting dependencies configuration and unmarshaling in defined structures
+	dat, err := ioutil.ReadFile("configuration/config.yml")
```

Comment:
Lets have this file in the project's root folder and call it `configuration.yml`